restock.lic

```diff
@@ -1,58 +1,65 @@
 #   Documentation: https://elanthipedia.play.net/Lich_script_repository#restock
-
 custom_require.call(%w[common common-items common-money])
-
 class Restock
   include DRC
   include DRCM
   include DRCI
-
   def initialize
     setup
-
     items = parse_restockable_items
+
+    custom_loc_items = items.select{ |k| k['hometown'] }
+
+    items -= custom_loc_items
+    restock_items(items)
+
+    unique_hometowns = custom_loc_items.map{ |k| "#{k['hometown']}" }.uniq
+    unique_hometowns.each do |hometown|
```

Comment:
💡 You don't need `unique_hometowns` as a variable. Method chaining here could look like
```ruby
custom_loc_items.map{ |k| "#{k['hometown']}" }
.uniq
.each do ...
```

Reply:
Yup, I used it differently before this, so I had a use of it then. Thanks!